week_1.py

```diff
@@ -0,0 +1,188 @@
+"""
+
+TASK 1
+
+Write a class to represent a Cash Register.
+You class should keep the state of price total and purchased items
+
+Below is a starter code:
+------------------------
+1. you can add new variables and function if you want to
+2. you will NEED to add accepted method parameters where required.
+For example, method add_item probably accepts some kind of an item?..
+3. You will need to write some examples of how your code works.
+
+"""
+
+class CashRegister:
+
+    def __init__(self):
+
+        self.total_items = dict() #None # {'item': 'price'}
+        self.total_price = 0
+        self.discount = 0 #in terms of percent
+
+    def add_item(self, item, price):
+        self.total_items[item] = price
+
+    def remove_item(self, item):
+        self.total_items.pop(item)
+        return self.total_items
+
+    def apply_discount(self, discount, item): #applied discount to the items
+        discount_item = int(self.total_items[item] * ((100-discount)/100))
```

Comment:
some more detailed variable names would be great also, instead of discount_item, this could be discounted_price, or discounted_item_price, just something more specific to what it is, as the discount_item could be a little confusing as it could be the actual item instead of the newly calculated price.